pkg/certificate/peer.go

```diff
@@ -74,9 +81,19 @@ func New(log *zap.Logger, ident *identity.FullIdentity, ca *identity.FullCertifi
 		}
 	}
 
-	peer.Certificates.AuthorizationDB = authorizationDB
-	peer.Certificates.Endpoint = NewEndpoint(log.Named("certificates"), ca, authorizationDB, uint16(config.MinDifficulty))
-	pb.RegisterCertificatesServer(peer.Server.GRPC(), peer.Certificates.Endpoint)
+	peer.AuthorizationDB = authorizationDB
+
+	peer.Certificate.Endpoint = NewEndpoint(log.Named("certificate"), ca, authorizationDB, uint16(config.MinDifficulty))
+	pb.RegisterCertificatesServer(peer.Server.GRPC(), peer.Certificate.Endpoint)
+
+	peer.Authorization.Listener, err = net.Listen("tcp", config.AuthorizationAddr)
```

Comment:
nit: Maybe we need validation (check if `config.AuthorizationAddr` is localhost) to avoid exposing endpoint by accident

Reply:
It isn't always necessarily the loopback interface though. It could be correct and desirable to bind to an private network interface.